modules/weko-records-ui/weko_records_ui/pdf.py

```diff
@@ -429,45 +445,57 @@ def get_oa_policy(activity_id):
             metadata_dict["affiliation"])
     ]
     metadata = '\n'.join(metadata_list)
-    metadata_lfnum = int(metadata.count('\n'))
-
-    for item in metadata_list:
-        metadata_lfnum += int(get_east_asian_width_count(item)
-                              ) // max_letters_num
 
     # Get url
     url = get_url(pid.pid_value)
-    url_lfnum = int(get_east_asian_width_count(url)) // max_letters_num
 
-    # Save top coordinate
-    top = pdf.y
     # Calculate x position of next cell
     offset = pdf.x + w1
+
+    # Save top coordinate
+    top = pdf.y
+    # Reset y coordinate
+    pdf.y = top
+    # Move to computed offset
+    pdf.x = offset
+    pdf.multi_cell(w2, meta_h, metadata, 1, 'L', False)
+    # Get height w2 and calculate line number of metadata
+    metadata_lfnum = 0
+    height = pdf.y - top
+    if height > 0:
```

Comment:
You can merge this line with the one above.

Reply:
It is a different treatment